[2025-09-27T00:00:09.832+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-27T00:00:09.840+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_forecast_stock_robust.create_or_replace_model scheduled__2025-09-26T00:00:00+00:00 [queued]>
[2025-09-27T00:00:09.842+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_forecast_stock_robust.create_or_replace_model scheduled__2025-09-26T00:00:00+00:00 [queued]>
[2025-09-27T00:00:09.842+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-09-27T00:00:09.846+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): create_or_replace_model> on 2025-09-26 00:00:00+00:00
[2025-09-27T00:00:09.848+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=27195) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-09-27T00:00:09.849+0000] {standard_task_runner.py:64} INFO - Started process 27197 to run task
[2025-09-27T00:00:09.849+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ml_forecast_stock_robust', 'create_or_replace_model', 'scheduled__2025-09-26T00:00:00+00:00', '--job-id', '420', '--raw', '--subdir', 'DAGS_FOLDER/ml_forecast_stock.py', '--cfg-path', '/tmp/tmplhb8uc98']
[2025-09-27T00:00:09.850+0000] {standard_task_runner.py:91} INFO - Job 420: Subtask create_or_replace_model
[2025-09-27T00:00:09.865+0000] {task_command.py:426} INFO - Running <TaskInstance: ml_forecast_stock_robust.create_or_replace_model scheduled__2025-09-26T00:00:00+00:00 [running]> on host 11e1994bdecf
[2025-09-27T00:00:09.892+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data226' AIRFLOW_CTX_DAG_ID='ml_forecast_stock_robust' AIRFLOW_CTX_TASK_ID='create_or_replace_model' AIRFLOW_CTX_EXECUTION_DATE='2025-09-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-26T00:00:00+00:00'
[2025-09-27T00:00:09.892+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-27T00:00:09.901+0000] {base.py:84} INFO - Using connection ID 'snowflake_default' for task execution.
[2025-09-27T00:00:09.902+0000] {connection.py:408} INFO - Snowflake Connector for Python Version: 3.11.0, Python Version: 3.12.4, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-09-27T00:00:09.902+0000] {connection.py:1258} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-09-27T00:00:11.552+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-09-27T00:00:12.064+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-09-27T00:00:13.091+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-09-27T00:00:13.092+0000] {ml_forecast_stock.py:223} INFO - Using: warehouse="BEETLE_QUERY_WH", database="ASH_DB", schema="STOCK_SCHEMA"
[2025-09-27T00:00:13.310+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-09-27T00:00:13.311+0000] {ml_forecast_stock.py:231} INFO - Executing SQL statement 1/1
[2025-09-27T00:04:00.244+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-09-27T00:04:00.421+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-09-27T00:04:00.422+0000] {ml_forecast_stock.py:237} INFO - All SQL statements executed successfully
[2025-09-27T00:04:00.422+0000] {connection.py:773} INFO - closed
[2025-09-27T00:04:00.555+0000] {connection.py:779} INFO - No async queries seem to be running, deleting session
[2025-09-27T00:04:00.946+0000] {ml_forecast_stock.py:306} INFO - ML Forecast model created successfully
[2025-09-27T00:04:00.948+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-27T00:04:00.949+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-27T00:04:00.970+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ml_forecast_stock_robust, task_id=create_or_replace_model, run_id=scheduled__2025-09-26T00:00:00+00:00, execution_date=20250926T000000, start_date=20250927T000009, end_date=20250927T000400
[2025-09-27T00:04:01.020+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-27T00:04:01.041+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-27T00:04:01.042+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
